A fix for issue #2877

JoeEylesBBC opened the pull request:
Changed the string used to identify incoming `eventStreams` in order to correctly handle multiple simultaneous events with different `value`s. fixing issue #2877.

nicosang commented:
Hi @JoeEylesBBC ,

thanks for your PR. Could you, please, update the unit test in order to be able to merge it?

Thanks again,
Nico

epiclabsDASH commented:
@JoeEylesBBC, given this is your first contribution and following dash.js [contribution guidelines](https://github.com/Dash-Industry-Forum/dash.js/blob/development/CONTRIBUTING.md), could you please also send the [feedback agreement](https://dashif.org/docs/DASH-IF-Feedback-Agreement-3-7-2014.pdf) document signed by you?

JoeEylesBBC commented:
Hi all,

Sorry for the late reply. It seems you should already have a signed CLA from the BBC R&D department - see https://github.com/Dash-Industry-Forum/dash.js/wiki/Meeting-Minutes#minutes-april-4th-2014

Joe

JoeEylesBBC commented:
Hi all,

I have just fixed a bug in which some events were not getting called and added it to this PR. This was due to the events being checked by a setInterval() every 0.1s, at which point the last 0.1s of events would be triggered; but the setInterval() would sometimes call after more than 0.1s, and thus any events older than 0.1s would not be triggered.
